PostsApp/src/Routes/postroutes.js

```diff
@@ -0,0 +1,83 @@
+var express = require('express');
+const postsRouter = express.Router();
+const { MongoClient,ObjectID } = require('mongodb');
+const debug = require('debug')('app:postRoutes');
+
+
+function router(nav) {
+   postsRouter.use((req,res,next)=>{
+    if(req.user){
+        next();
+      }else{
+        res.redirect('/');
+      }
+   });
+
+    postsRouter.route('/')
+        .get((req, res) => {
+
+            const url = 'mongodb://localhost:27017';
```

Comment:
instead of using the details like url and db name, user, password etc.. put them in a config file and import where they are needed.

never repeat !!

Reply:
made a separate file named dbconfig.js and used to import details like url, db name. Hence made the suggested changes.